autogpt_platform/frontend/CONTRIBUTING.md

```diff
@@ -0,0 +1,765 @@
+<div align="center">
+  <h1>AutoGPT Frontend • Contributing ⌨️</h1>
+  <p>Next.js App Router • Client-first • Type-safe generated API hooks • Tailwind + shadcn/ui</p>
```

Comment:
Consider putting this in the docs/ as a link in the mkdocs.yaml

Reply:
I'll do on a follow up PR 👍🏽